cmd/ks-apiserver/app/server.go

```diff
@@ -73,6 +74,17 @@ cluster's shared state through which all other components interact.`,
 		fmt.Fprintf(cmd.OutOrStdout(), "%s\n\n"+usageFmt, cmd.Long, cmd.UseLine())
 		cliflag.PrintSections(cmd.OutOrStdout(), namedFlagSets, cols)
 	})
+
+	versionCmd := &cobra.Command{
+		Use:   "version",
+		Short: "Print the version of KubeSphere ks-apiserver",
+		Run: func(cmd *cobra.Command, args []string) {
+			fmt.Println(version.Get())
```

Comment:
It would be easy to redirect the output to a different place if using `cmd.Println` instead of `fmt.Println`

Reply:
yeah, didn't know there is a `cobra.Println`, fixed.